tests/reftests/update-upgrade.test

```diff
@@ -506,3 +506,56 @@ The following actions will be performed:
 -> removed   ongoing.dev
 -> installed ongoing.dev
 Done.
+### <REPO/packages/d-foo/d-foo.6/opam>
+opam-version: "2.0"
+build: [ "false" ]
+### opam update
+[NOTE] ongoing.dev has previously been updated with --working-dir, not resetting unless explicitly selected
+
+<><> Updating package repositories ><><><><><><><><><><><><><><><><><><><><><><>
+[second] no changes from file://${BASEDIR}/REPO2
+[default] synchronised from file://${BASEDIR}/REPO
+
+<><> Synchronising development packages <><><><><><><><><><><><><><><><><><><><>
+[quux.dev5] synchronised (no changes)
+Now run 'opam upgrade' to apply any package updates.
```

Comment:
Is this `opam update` output needed?

Reply:
I think currently yes but we could remove it if the new version of `d-foo.6` was introduced using `<pkg:d-foo.6>` instead of `<REPO/...>`